src/app/components/pages/IsaacCompetition/EntryForm/CompetitionEntryForm.tsx

```diff
@@ -54,19 +60,23 @@ const CompetitionEntryForm = ({ handleTermsClick }: CompetitionEntryFormProps) =
                   label="First Name"
                   type="text"
                   id="firstName"
+                  required={true}
```

Comment:
small tip, you don't need to write `true` in a prop when it's true. Just declaring the prop works, like so: 

```jsx
<Component 
  id="firstName"
  required
  disabled
/>
```